### PR TITLE
Ball destroying

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

**Review by mwawrzen — APPROVED** [review]
LGTM

File: models/objects/chain_bullet.py [hunk 1]
```diff
@@ -26,6 +28,26 @@ def __init__(self, position: Vector2, space):
...
+        for s in data.shapes:
+            if s.collision_type == ObjectCollisionType.CHAIN:
+                for obj in self.chains_elements:
+                    shape = list(obj.body.shapes)[0]
+                    if shape is s:
+                        self.delete_self()
+                        break
+
+            if s.collision_type == ObjectCollisionType.BALL:
+                for obj in models.game_manager.GameManager().scene.objects:
```
Comment (jaanonim) [inline models/objects/chain_bullet.py]: This is strange. Maybe import `GameManager` instead of whole file

File: models/objects/chain_bullet.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import time
 from pygame import Vector2, Surface
+
+import models.game_manager
```
Comment (jaanonim) [inline models/objects/chain_bullet.py]: .

**Review by jaanonim — CHANGES_REQUESTED** [review]
Weird import

**Review by jaanonim — APPROVED** [review]

**Pull request merged**